friendica/content.md

```diff
@@ -0,0 +1,290 @@
+# What is Friendica?
+
+Friendica is a decentralised communications platform that integrates social communication. Our platform links to independent social projects and corporate services.
+
+![logo](https://cdn.rawgit.com/friendica/docker/9c954f4d/friendica.svg)
+
+# How to use this image
+
+The images are designed to be used in a micro-service environment. There are two types of the image you can choose from.
+
+The `apache` tag contains a full Friendica installation including an apache web server. It is designed to be easy to use and gets you running pretty fast. This is also the default for the `latest` tag and version tags that are not further specified.
+
+The second option is a `fpm` container. It is based on the [php-fpm](https://hub.docker.com/_/php/) image and runs a fastCGI-Process that serves your Friendica server. To use this image it must be combined with any Webserver that can proxy the http requests to the FastCGI-port of the container.
+
+[![Try in PWD](https://github.com/play-with-docker/stacks/raw/cff22438cb4195ace27f9b15784bbb497047afa7/assets/images/button.png)](http://play-with-docker.com?stack=https://raw.githubusercontent.com/nupplaphil/friendica-docker/fec33c98be957436279b7074ca08068b18622627/stack.yml) (Admin-E-Mail: `root@friendica.local`)
+
+## Using the apache image
+
+You need at least one other mariadb/mysql-container to link it to Friendica.
+
+The apache image contains a webserver and exposes port 80. To start the container type:
+
+```console
+$ docker run -d -p 8080:80 --link some-mysql:mysql friendica/server
```

Comment:
```suggestion
$ docker run -d -p 8080:80 --link some-mysql:mysql %%IMAGE%%
```

You probably want to take a look at https://github.com/docker-library/docs/pull/1441 for how we've updated most other image documentation to avoid the soft-deprecated `--link` option (although whether to keep that or not is ultimately up to you).